src/testers.jl

```diff
@@ -142,6 +170,8 @@ function rrule_test(f, ȳ, xx̄s::Tuple{Any, Any}...; rtol::Real=1e-9, atol::Re
     y = f(xs...; fkwargs...)
     check_equal(y_ad, y; isapprox_kwargs...)  # make sure primal is correct
 
+    ȳ = tangent(auto_primal_and_tangent(y ⊢ output_tangent))
```

Comment:
It's a bit stilly to be attaching a tangent to the primal explicitly, then pealing it off, but it makes both given and  `Auto()` work

Reply:
the alternative is:
```suggestion
    ȳ = output_tangent === Auto() ? rand_tangent(primal) : output_tangent
```

Which maybe is cleaner?

idk yesterday I was hating on branches.
I started to write `auto_tangent(primal, output_tangent)` and realized that it overlapped with `auto_primal_and_tangent`

Reply:
`ȳ = output_tangent === Auto() ? rand_tangent(primal) : output_tangent` is certainly much clearer, to my eyes